### PR TITLE
Updated code repo to accompany tutorial

### PR DESCRIPTION
**Pull request by duplo83**

### CODE REVIEW

**Comment by judithpatudith** [comment]
@duplo83 thanks for this! Please request reviews from @hashicorp/terraform-education so that the team can help out too :)

**Comment by duplo83** [comment]
I can't seem to find that group, I'm seeing hashicorp/consul-education, but not terraform.

**Comment by judithpatudith** [comment]
Good catch! Just added it to the repo! thanks!

File: terraform.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+terraform {
+    /* Uncomment this block to use Terraform Cloud for this tutorial
```
Comment (ritsok) [inline terraform.tf]:
```suggestion
/* Uncomment this block to use Terraform Cloud for this tutorial
```

File: terraform.tf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+cloud {
+    organization = "organization-name"
+    workspaces {
+      name = "learn-terraform-REPLACE_ME"
```
Comment (ritsok) [inline terraform.tf]:
```suggestion
      name = "learn-terraform-sensitive-variables"
```

**Pull request merged**